west.yml

```diff
@@ -33,6 +33,14 @@ manifest:
       url-base: https://github.com/nanopb
     - name: alexa
       url-base: https://github.com/alexa
+    - name: oyro
```

Comment:
Half of these remotes are not used. And anyway, is it really good for master? Do we want to introduce some custom forks for the repositories? Note, once added the forks cannot be deleted, otherwise old revisions of NCS will not be able to fetch all modules.

Reply:
This was supposed to be changed today, but I have added the DNM. Will clarify in the PR description, sorry for time wasted :)

Reply:
Sure, I noticed later that GH bot messed with your DNM:
![image](https://user-images.githubusercontent.com/22167799/102508525-c4b5aa00-4085-11eb-97a9-31091da8e3fb.png)